maestrowf/datastructures/yamlspecification.py

```diff
@@ -148,8 +162,9 @@ def verify(self):
         self.verify_study()
         self.verify_parameters()
```

Comment:
Noticed when I fixed the schema that it still wasn't checking the environment. Found that this call was missing.

```suggestion
        self.verify_environment()
        self.verify_study()
        self.verify_parameters()
```

Reply:
Holy cow ! Good catch